src/org/gk/slicing/SlicingEngine.java

```diff
@@ -161,6 +182,47 @@ public void setTargetDbPort(int dbPort) {
         this.targetDbPort = dbPort;
     }
     
+   /**
+     * The name of the compare database. This database will be created at the same host
+     * as the data source.
+     * @param DbName
+     */
+    public void setCompareDbName(String DbName) {
+        this.compareDbName = DbName;
+    }
+
+    public void setCompareDbHost(String host) {
+        this.compareDbHost = host;
+    }
+
+    public void setCompareDbUser(String dbUser) {
+        this.compareDbUser = dbUser;
+    }
+
+    public void setCompareDbPwd(String pwd) {
+        this.compareDbPwd = pwd;
+    }
+
+    public void setCompareDbPort(int dbPort) {
+        this.compareDbPort = dbPort;
+    }
+
+    public void initCompareDBA() throws SQLException {
+    	this.compareDBA = new MySQLAdaptor(compareDbHost,
+										   compareDbName,
+										   compareDbUser,
+										   compareDbPwd,
+										   compareDbPort);
+    }
+
+    public void setCompareClasses(List<String> compareClasses) {
```

Comment:
No need. See comments for slicingTool.prop.

Reply:
Fixed in [5d0061c](https://github.com/reactome/CuratorTool/pull/20/commits/5d0061cf2e6db424efa382db7c7d1255250b379b).